src/main/java/org/c4sg/service/impl/OrganizationServiceImpl.java

```diff
@@ -88,4 +106,21 @@ public String getLogoUploadPath(Integer organizationId) {
         return LOGO_UPLOAD.getValue() + File.separator + organizationId + IMAGE.getValue();
     }
 
+	@Override
+	public List<OrganizationDTO> findByUser(Integer userId) {
+		User user = userDAO.findById(userId);
+        requireNonNull(user, "Invalid User Id");
+		List<UserOrganization> userOrganizations = userOrganizationDAO.findByUserId(userId);
+		Collections.sort(userOrganizations, (p1, p2) -> {
+			int result = 0;
+			result = p1.getStatus().compareTo(p2.getStatus());
+			return result*-1;
```

Comment:
I found this piece of code with -1, we need to refactor this in the future
`   @Override
    public List<ProjectDTO> findByUser(Integer userId) {
        User user = userDAO.findById(userId);
        requireNonNull(user, "Invalid User Id");
        List<UserProject> userProjects = userProjectDAO.findByUserId(userId);
        Comparator<UserProject> projectComp = new Comparator<UserProject>() {
            @Override
            public int compare(UserProject o1, UserProject o2) {
                int result = 0;
                // result = o1.getStatus().compareTo(o2.getStatus());
                return result * -1;
            }
        };
        Collections.sort(userProjects, projectComp);
        List<ProjectDTO> projectDtos = new ArrayList<ProjectDTO>();
        for (UserProject userProject : userProjects) {
            projectDtos.add(projectMapper.getProjectDtoFromEntity(userProject));
        }
        return projectDtos;
    }`